src/app/global.ts

```diff
@@ -0,0 +1,12 @@
+export interface IGlobal {
```

Comment:
Great use of interface (and naming it with the capital I before). 

Heads-up here: always declare the mandatory properties before the optional(?) ones. (for future reference)